### ISSUE TITLE
Convert world space matrix to parent space

### ISSUE THREAD

**Issue by takahirox** [takahirox/glTF-Blender-IO-EXT-mesh-gpu-instancing#6]
https://github.com/takahirox/glTF-Blender-IO-EXT-mesh-gpu-instancing/blob/5ae881c067310abff2f3dd70f21f1c579d6eeb4c/__init__.py#L70-L72

**Comment by Suprhimp** [comment]
Hi, can I ask how long does it takes to fix this problem?

**Comment by Suprhimp** [comment]
https://github.com/mrdoob/three.js/issues/24968#issuecomment-1317936919

As [WestLangley](https://github.com/WestLangley) saids, I found that the parent matrix applied twice.

I forcely changed the node's matrix as default, I can check it reveals normally.

```
            gltf2_node.mesh = result
            gltf2_node.rotation = [0.0,0.0,0.0,1.0]
            gltf2_node.translation = [0.0,0.0,0.0]
            gltf2_node.scale = [1.0,1.0,1.0]
```

But you know, it is not good solution, and also, if I add animation of instanced mesh, it goes wrong coordinate again.

**Comment by takahirox** [comment]
> Hi, can I ask how long does it takes to fix this problem?

Soon if I can start to work on it because probably it is easy to fix. But I just don't have time right now. Making PR is very welcome.

**Comment by takahirox** [comment]
Merged #6

Do you mind testing with the latest source code to check if instance attributes are in local space?

**Comment by Suprhimp** [comment]
still..
I can give you my model
[instTest.blend.zip](https://github.com/takahirox/glTF-Blender-IO-EXT-mesh-gpu-instancing/files/10087879/instTest.blend.zip)

**Comment by takahirox** [comment]
I think the problem has been fixed with #9

**Comment by Suprhimp** [comment]
Thanks, It's working!
